cmd/testgrid-config-generator/main.go

```diff
@@ -171,6 +195,11 @@ func main() {
 				blocking.add(job.ProwJob.Name)
 			}
 		}
+		for _, p := range informingPostSubmits {
+			if p.Labels["job-release"] == version {
```

Comment:
We likely want a presubmit in `release` that enforces that you set this if you set it to be informing, and maybe also if you commit to release-controller config that you set the infroming/blocking label, too?

Reply:
Agree